compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -8663,7 +8666,7 @@ TR::Node *dmulSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
       {
       // In IEEE FP Arithmetic, d * 1.0 is d
       // Pretend to be an int before doing the comparison
-      auto binOpSimplifier = getLongBinaryOpSimplifier(s);
+      auto binOpSimplifier = getDoubleBitsBinaryOpSimplifier(s);
       auto identity = binOpSimplifier.tryAndSimplifyIdentityOp(node, (int64_t) DOUBLE_ONE);
```

Comment:
I don't think `DOUBLE_ONE` should be cast to `int64_t` any more

Reply:
Good point :eyes: